### ISSUE TITLE
Languages translated but not available 

### ISSUE THREAD

**Issue by tmassimi** [goauthentik/authentik#11550]
**Describe the bug**
setting my browser in french, I see authentik in french. setting italian, I see authentik is in english

but italian translation is present on this repo.

this file says that is necessary to run lit-localize to update the language list

https://github.com/goauthentik/authentik/blob/main/web/src/locale-codes.ts

i guess this should be done before each new release is published

**Comment by nmersi** [comment]
I have the same problem. I need Italian but also if I select another language for example french I see authentik in english.
I'm using the docker installation. ghcr.io/authentik/server:2024.8.3.

**Comment by tmassimi** [comment]
finally I have a trace to follow, found in another bug:

"BeryJu commented last week
The translations will automatically be merged into authentik once they reach 100% on transifex, at which point they will also be shown as an option in the user interface"

so it is required to transalte at this link:

https://explore.transifex.com/authentik/authentik/

**Comment by nmersi** [comment]
Thank you for the information.
I registered for the translation, but I think some other did because in few days translation in Italian is 100%
Do somebody know if 100% translation is sufficient? Because also 100% is not reviewed. 
And I have another question: How can I find if it will be in next docker image?

**Comment by tmassimi** [comment]
ciao Nicola, I have your same question... 
BTW: with this commit the russian language has been added

https://github.com/goauthentik/authentik/commit/fea79dd1201db237df03e818adaa563fc578334b

probably we should do the same things, these are the instruction:
 
https://docs.goauthentik.io/docs/developer-docs/translation
